File: oneday/scripts1/info.c
#include <unistd.h>
#include <sys/types.h>
#include <stdio.h>
#include <stdlib.h>
main() {
 char name[128],fname[128];
 pid_t mypid;
 FILE *f;

/* get the process id */
mypid=getpid();
/* get the host name */
gethostname(name, 128);
/* make a file name based on these two */
sprintf(fname,"%s_%8.8d",name,(int)mypid);
/* open and write to the file */
f=fopen(fname,"w");
fprintf(f,"C says hello from %d on %s\n",(int)mypid,name);
}

File: oneday/scripts1/info.py
#!/opt/development/python/2.6.5/bin/python
import os
# get the process id
mypid=os.getpid()
# get the node name
name=os.uname()[1]
# make a filename based on the two
fname="%s_%8.8d" % (name,mypid)
# open and write to the file
f=open(fname,"w")
f.write("Python says hello from %d on %s\n" %(mypid,name))

